pupil-spa/src/app/submission-pending/submission-pending.component.ts

```diff
@@ -29,6 +28,7 @@ export class SubmissionPendingComponent implements OnInit, AfterViewInit, OnDest
               private questionService: QuestionService,
               private speechService: SpeechService,
               private checkStatusService: CheckStatusService,
+              private checkCompleteService: CheckCompleteService,
```

Comment:
It seems like `StorageService`, `SubmissionService` and `AuditService` are not needed anymore (& their imports at the top)?

Reply:
@danwilliams removed